.gitattributes

```
@@ -0,0 +1,34 @@
+# Make packagist / composer download smaller
+/.ddev export-ignore
+/.github export-ignore
+/dev
```

```suggestion
/dev
/docs
```

what do you think about adding the /docs folder?

---

Yeah, no one needs docs inside vendor